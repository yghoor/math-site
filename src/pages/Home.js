const Home = () => (
  <div className="home-page">
    <h1>Welcome to our page!</h1>

    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt,
      possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor
      ipsa quo commodi nulla? Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet
      quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis
      dolore ipsam ut dolor ipsa quo commodi nulla? Lorem ipsum dolor sit amet consectetur
      adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur
      quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla?
    </p>

    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt,
      possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor
      ipsa quo commodi nulla? Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet
      quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis
      dolore ipsam ut dolor ipsa quo commodi nulla? Lorem ipsum dolor sit amet consectetur
      adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur
      quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla?
    </p>
  </div>
);

export default Home;