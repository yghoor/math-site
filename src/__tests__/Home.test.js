/* eslint-disable no-unused-vars */
import renderer from 'react-test-renderer';

import Home from '../pages/Home.js';

test('renders "Home" page', () => {
  const tree = renderer
    .create(<Home />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});