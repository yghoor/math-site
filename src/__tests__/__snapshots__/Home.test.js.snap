// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders "Home" page 1`] = `
<div
  className="home-page"
>
  <h1>
    Welcome to our page!
  </h1>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla? Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla? Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla?
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla? Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla? Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla?
  </p>
</div>
`;

exports[`renders Home page 1`] = `
<div
  className="home-page"
>
  <h1>
    Welcome to our page!
  </h1>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla? Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla? Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla?
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla? Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla? Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum amet quod fugit incidunt, possimus ea voluptatibus. Error tenetur quis ullam minus, perferendis dolore ipsam ut dolor ipsa quo commodi nulla?
  </p>
</div>
`;
